static/css/floorplan.css

```diff
@@ -29,6 +29,8 @@
   color: white;
   cursor: pointer;
   user-select: none;
+  touch-callout: none;
+  -webkit-touch-callout: none;
```

Comment:
Huh, I didn't know this existed.

Reply:
Yeah, it's apparently only an iOS thing but I threw it in there just in case